riak_test/src/rtcs.erl

```diff
@@ -1191,3 +1191,18 @@ get_cmd_result(Port, WaitTime) ->
     after WaitTime ->
             {error, timeout}
     end.
+
+    %% Copy from rts:iso8601/1
+iso8601(Timestamp) when is_integer(Timestamp) ->
+    GregSec = Timestamp + 719528 * 86400,
+    Datetime = calendar:gregorian_seconds_to_datetime(GregSec),
+    {{Y,M,D},{H,I,S}} = Datetime,
+    io_lib:format("~4..0b~2..0b~2..0bT~2..0b~2..0b~2..0bZ",
+                  [Y, M, D, H, I, S]).
+
+reset_log(CSNode) ->
```

Comment:
`s/CSNode/Node/` because this function can be used for riak/cs/stanchion in the same way.